lib/constants/common.ts

```diff
@@ -0,0 +1,12 @@
+export const MILLISECONDS_PER_SECOND = 1_000;
```

Comment:
Why `1_000` not `1000`?

Reply:
This notation is called numeric separator in typescript 2.7

https://mariusschulz.com/blog/numeric-separators-in-typescript